.github/workflows/tag-and-release.yml

```diff
@@ -0,0 +1,53 @@
+---
+name: Tag & Release
+'on':
+  push:
```

Comment:
Should we do the usual workflow_call pattern here?

Reply:
Good catch. Otherwise we can't use it as a reusable workflow. This has been fixed in 7c562e4.

Reply:
Do we want the push event in this one? Usually the reusable workflows just have workflow_call